apps/toys/ColorCube/.project

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
 	<name>ColorCube</name>
+	<comment></comment>
```

Comment:
This change doesn't do anything as far as I know. It can be reverted. It can also stay.